### DISCUSSION TITLE
Image Generation models for Pro Users

### DISCUSSION THREAD

**Discussion by muralidhar972** [perplexityai/api-discussion#28] [General]
hello team, 

I Need small help in understanding below message and below answers below questions. 

1.As per the your subscription information Pro users can choose models like GPT-4o and others. If yes how can I use them using API.  
2."Visualize answers using Playground AI, DALL-E and others." Can we use this using API. 

Please add more details if i missed anything. attached screen shot for reference

![Screenshot 2024-09-10 093058](https://github.com/user-attachments/assets/be1f3023-573f-4fe9-99f3-d409e0f0ff57)

Regards.

**Comment by rafs2006** [comment]
Hey @muralidhar972! Here are the models available via our API: https://docs.perplexity.ai/guides/model-cards
Image generation is available on the desktop version of the website, not via API.
